src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IAnonymousFunctionExpression.cs

```diff
@@ -753,24 +753,91 @@ void M(System.Action d1, System.Action<bool, bool> d2)
             var graphM = ControlFlowGraph.Create((IMethodBodyOperation)semanticModel.GetOperation(tree.GetRoot().DescendantNodes().OfType<MethodDeclarationSyntax>().Single()));
 
             Assert.NotNull(graphM);
+            Assert.Null(graphM.Parent);
 
             IFlowAnonymousFunctionOperation lambdaD1 = getLambda(graphM);
 
             Assert.Throws<ArgumentNullException>(() => graphM.GetLocalFunctionControlFlowGraph(null));
             Assert.Throws<ArgumentOutOfRangeException>(() => graphM.GetLocalFunctionControlFlowGraph(lambdaD1.Symbol));
+            Assert.Throws<ArgumentNullException>(() => graphM.GetLocalFunctionControlFlowGraphInScope(null));
+            Assert.Throws<ArgumentOutOfRangeException>(() => graphM.GetLocalFunctionControlFlowGraphInScope(lambdaD1.Symbol));
 
             var graphD1 = graphM.GetAnonymousFunctionControlFlowGraph(lambdaD1);
             Assert.NotNull(graphD1);
+            Assert.Same(graphM, graphD1.Parent);
```

Comment:
Consider adding a similar set of assertions for local functions.

Reply:
`LocalFunctionFlow_11`

---
In reply to: [223793329](https://github.com/dotnet/roslyn/pull/30395#discussion_r223793329) [](ancestors = 223793329)